integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -76,22 +76,16 @@ def export(self, **kwargs):
         Return the exported and transformed content metadata as a dictionary.
         """
         content_metadata_export = {}
-        if can_use_enterprise_catalog(self.enterprise_customer.uuid):
-            content_metadata_items = self.enterprise_catalog_api.get_content_metadata(
-                self.enterprise_customer,
-                enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
-            )
-            LOGGER.info(
-                'Getting metadata for Enterprise [%s], Catalogs [%s] from Enterprise Catalog Service. Results: [%s]',
-                self.enterprise_customer.name,
-                self.enterprise_configuration.customer_catalogs_to_transmit,
-                json.dumps(content_metadata_items)
-            )
-        else:
-            content_metadata_items = self.enterprise_api.get_content_metadata(
-                self.enterprise_customer,
-                enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
-            )
+        content_metadata_items = self.enterprise_catalog_api.get_content_metadata(
+            self.enterprise_customer,
+            enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
+        )
+        LOGGER.info(
+            'Getting metadata for Enterprise [%s], Catalogs [%s] from Enterprise Catalog Service. Results: [%s]',
+            self.enterprise_customer.name,
+            self.enterprise_configuration.customer_catalogs_to_transmit,
+            json.dumps(content_metadata_items)
+        )
         LOGGER.info('Retrieved content metadata for enterprise [%s]', self.enterprise_customer.name)
```

Comment:
Do we need both of these logs? We could probably remove the `LOGGER.info` on line 89.